### PR TITLE
Update Readme to clearify intent of project and remove outdated specs

### PR DESCRIPTION
**Pull request by andrerom**
simplifies spec and project goals to focus on the high level needs instead of low level details of how it was thought (half a year ago) that they would be accomplished.

### CODE REVIEW

**Comment by andrerom** [comment]
review ping @vidarl

File: README.md [hunk 1]
```diff
@@ -1,134 +1,49 @@
...
-- Clustering:
- - Cache: Memcached|Redis|..
- - HTTP cache: Varnish|Nginx|..
- - FS: NFS|GridFS|..  (services like S3 might not need a container, but could have a container acting as proxy)
+### Spec
 
-**Note** We should take advantage of official images from Docker as much as possible, however they now use Debian
-as it takes less space then Ubuntu, and for lowest space use (and memory?) base images should be the same across
-all our official images.
+All Docker service containers, aka micro services, like Nginx, Apache, Mysql, Varnish, Solr and so on should have no direct knowledge of eZ Platform.
```
Comment (vidarl) [inline README.md]: This is desirable, yes. But also quite difficult.
For instance, ezp's virtualhost must be injectected into nginx. Okay, can be solved by using a volume where config is stored.
However, if using varnish, you must:
 - inject varnish' IP into nginx config ( must set TRUSTED_PROXIES etc in nginx config )
 - inject nginx' IP into varnish' confing
 - inject .VCL into varnish
 - inject varnish IP etc in ezpublish.conf ( configure purge servers etc )

And this must be done *every* time the varnish or nginx container (re)starts as the IPs are different everytime....

Comment (andrerom) [inline README.md]: updated the text a bit

**Comment by vidarl** [comment]
+1

File: README.md [hunk 2]
```diff
@@ -1,134 +1,50 @@
...
- - HTTP cache: Varnish|Nginx|..
- - FS: NFS|GridFS|..  (services like S3 might not need a container, but could have a container acting as proxy)
+### Spec
 
-**Note** We should take advantage of official images from Docker as much as possible, however they now use Debian
-as it takes less space then Ubuntu, and for lowest space use (and memory?) base images should be the same across
-all our official images.
+All Docker service containers, aka micro services, like Nginx, Apache, Mysql, Varnish, Solr and so on should ideally have no direct knowledge of eZ Platform.
+Injection of configuration should be done at startup by either passing env variables and/or mounting generic configuration files.
+The motivation for this is to be able to contribute our genric extensibility needs upstream to official docker containers.
```
Comment (joaoinacio) [inline README.md]: `s/genric/generic/`

**Pull request merged**